src/openapi.yaml

```diff
@@ -324,6 +384,111 @@ components:
             format: uri
           readOnly: true
           uniqueItems: true
+        eigenschappen:
+          title: Eigenschappen
+          type: array
+          items:
+            type: string
+            format: uri
+          readOnly: true
+          uniqueItems: true
+    EigenschapSpecificatie:
+      title: Specificatie
+      required:
+        - formaat
+        - lengte
+        - kardinaliteit
+      type: object
+      properties:
+        groep:
+          title: Groep
+          description: >-
+            Benaming van het object of groepattribuut waarvan de EIGENSCHAP een
+            inhoudelijk gegeven specificeert.
+          type: string
+          maxLength: 32
+        formaat:
+          title: Formaat
+          description: >-
+            Het soort tekens waarmee waarden van de EIGENSCHAP kunnen worden
+            vastgelegd.
+          type: string
+          enum:
+            - tekst
+            - getal
+            - datum (jjjjmmdd)
+            - datum/tijd (jjjjmmdduummss)
+        lengte:
+          title: Lengte
+          description: >-
+            Het aantal karakters (lengte) waarmee waarden van de EIGENSCHAP
+            worden vastgelegd.
+          type: string
+          maxLength: 14
+        kardinaliteit:
+          title: Kardinaliteit
+          description: >-
+            Het aantal mogelijke voorkomens van waarden van deze EIGENSCHAP bij
```

Comment:
Wellicht moeten de validatieregels ook in de description. Het is onduidelijk dat hier nu 0, 1, 999 en N kan komen te staan. Misschien kunnen we de validators wel uit het model halen en automatisch in de description proppen?

Reply:
Denk dat dit met (unit) tests kan verduidelijkt worden in eerste instantie. Op dit moment zijn de endpoints read-only, maar bij schrijfacties zou dit in de foutmeldingen kunnen terecht komen. Is wel een open discussiepunt hoe ver de (mogelijke) fouten gedocumenteerd worden in de openapi spec